docs/docs/15-concepts.md

```diff
@@ -1,9 +1,8 @@
 ---
-title: Concepts
-description: Concepts
+sidebar_label: Key Concepts
+description: Find out more about key Kargo concepts - stage, freight, promotion, subscription, and more
```

Comment:
```suggestion
description: Find out more about key Kargo concepts - stages, freight, warehouses, promotions, and more
```

s/subscriptions/warehouses/ + reorder to match the order in which the concepts are presented.